action/protocol/staking/protocol.go

```diff
@@ -433,22 +453,29 @@ func (p *Protocol) handle(ctx context.Context, act action.Action, csm CandidateS
 		rLog, tLogs, err = p.handleCandidateEndorsement(ctx, act, csm)
 	case *action.CandidateTransferOwnership:
 		rLog, tLogs, err = p.handleCandidateTransferOwnership(ctx, act, csm)
+	case *action.MigrateStake:
+		var nonceUpdated bool
+		logs, tLogs, gasConsumed, nonceUpdated, err = p.handleStakeMigrate(ctx, act, csm)
```

Comment:
why not follow other actions `rLog, tLogs ... =`

Reply:
`rLog` can only represent log for one address, whereas here it will include logs for multiple addresses (i.e. NativeStakingAddress and ContractStakingAddress).